### ISSUE TITLE
Black Screen!

### ISSUE THREAD

**Issue by MoudBarthez** [Muirey03/SmallSiri#29]
siri still works and the small siri appears, but it blacks out the rest of the screen!

**Comment by GoldminorguyProSkilzz** [comment]
iPhone 6/6 Plus?

**Comment by MoudBarthez** [comment]
Oh yes iPhone 6

**Comment by GoldminorguyProSkilzz** [comment]
Im getting this problem as well!

**Comment by MoudBarthez** [comment]
Yes, have you had any freezes recently, i kept experiencing freezes and restarts in the last few days

**Comment by GoldminorguyProSkilzz** [comment]
Not that I know of, since i deleted Small Siri when the black screen glitch happened

**Comment by GoldminorguyProSkilzz** [comment]
people are also experiencing this bug [here](https://github.com/Muirey03/SmallSiri/issues/22)
